src/finchlite/finch_assembly/interpreter.py

```diff
@@ -335,34 +335,17 @@ def my_func(*args_e):
                             )
                 return AssemblyInterpreterModule(self, kernels)
             case asm.Print(args):
-                match args:
-                    case asm.Variable():
-                        print(f"{self(args)}")
-                    case _:
-                        if isinstance(args, tuple):
-                            arg_decls = [
-                                f"{self(arg)}"
-                                for arg in args
-                                if isinstance(arg, asm.Variable)
-                            ]
-                            print(arg_decls)
-                        else:
-                            raise NotImplementedError(
-                                f"Unrecognized argument type: {args}"
-                            )
-                return None
-            case asm.Debug(message, args):
                 match args:
                     case asm.Variable(name, _):
-                        print(f"{message} " + f"{name}=" + str(self(args)))
+                        print(f"{name}=" + str(self(args)))
```

Comment:
Don't add the name, you can just print the value. Make it behave like Print(args) = print(cat(args))